druid-shell/src/windows/mod.rs

```diff
@@ -1284,6 +1286,11 @@ impl<'a> WinCtx<'a> for WinCtxImpl<'a> {
             .unwrap_or(0);
         TimerToken::new(id)
     }
+
+    fn set_clipboard_contents(&mut self, contents: ClipboardItem) {
+        log::info!("set_clipboard_contents item: {:?}", &contents);
```

Comment:
Not sure we should be keeping this log, though I certainly see how it's helpful for development.